public/static/partials/user_volume_options.html

```diff
@@ -0,0 +1,25 @@
+<form name="form" novalidate ng-submit="form.$valid && createVolume()">
+  <fieldset>
+    <div style="margin-left: 10px;">
+      <div style="margin-bottom:5px">
+        <label> Volume size (GB) </label>
+        <input type="number" ng-model="createVolumeInfo.size" min="1"/>
+        <span class="warning-text" ng-show="invalidVolumeSize(createVolumeInfo.size)"><i class="fa fa-warning"></i>
+      Can have at most [[ maxVolumeSizePerUser ]] GiB per user
+        </span>
+      </div>
+      <div style="margin-bottom:5px">
+        <label> Availability Zone</label>
+        <input type="text" id="input-zone"  ng-model="createVolumeInfo.zone"/>
+      </div>
+      <div style="margin-bottom:5px">
+        <label> Type </label>
+        <input type="text" id="input-type"  ng-model="createVolumeInfo.type"/>
+      </div>
+    </div>
+    <div>
+      <button type="submit" class="btn btn-primary" style="float: left; margin-left: 10px;" ng-disabled="!form.$valid || invalidVolumeSize(createVolumeInfo.size)">Spawn</button>
```

Comment:
if a user double clicks the button will it make two volumes? If so, should there be a mechanism to prevent it?

Reply:
They shouldn't be able to... but I'll re-add the thing.